Plugins/Public/mobiledocking_plugin/Main.h

```diff
@@ -38,18 +41,26 @@ struct CLIENT_DATA
 	bool baseUndock = false;
 };
 
-struct DEFERREDJUMPS
+struct ActionJettison
 {
-	uint system;
-	Vector pos;
-	Matrix rot;
+	int timeLeft;
+	wstring carrierCharname;
+	wstring dockedCharname;
 };
 
-static map<uint, DEFERREDJUMPS> mapDeferredJumps;
+extern vector<ActionJettison> jettisonList;
 
-void LoadShip(string shipFileName);
-void SaveDockInfoCarrier(const wstring& shipFileName, uint clientID, const CLIENT_DATA& client);
-void SaveDockInfoCarried(const wstring& shipFileName, uint clientID, const CLIENT_DATA& client);
+struct DelayedJettison

```

Comment:
why is this function in a struct?

Reply:
I'd like to think that I create an object.

Reply:
Why?

Reply:
Really, why? Now function is extracted from struct.